swerve_controller/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(swerve_controller)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
```

Comment:
in melodic we have this by default, please remove

Reply:
Will do

Reply:
It's copy-paste from the four_wheel_steering_controller btw